lib/providers/services/links_service.dart

```diff
@@ -63,4 +63,9 @@ class LinksService {
 
     return dynamicUrl;
   }
+
+  Future<Uri> unpackDynamicLink(String link) => FirebaseDynamicLinks.instance
+    .getDynamicLink(Uri.parse(link))
+    .then((PendingDynamicLinkData dynamicLink) => dynamicLink.link);
+
 }
```

Comment:
Can we use this everywhere we process dynamic links? E.g. the invite link too?

Reply:
Sure!